Riot/Modules/SetPinCode/SetPinCoordinator.swift

```diff
@@ -27,7 +27,11 @@ final class SetPinCoordinator: SetPinCoordinatorType {
     
     private let navigationRouter: NavigationRouterType
     private let session: MXSession?
-    private var viewMode: SetPinCoordinatorViewMode
+    var viewMode: SetPinCoordinatorViewMode {
```

Comment:
Can we use a dedicated method `func update(viewMode: SetPinCoordinatorViewMode)`?
The `start` method could use it. I do not like to have a `start` method that can be called several times.

Reply:
I changed to have a private `updateRootCoordinator` method.